example/NEMA/NEMA_H0/main.py

```diff
@@ -237,12 +242,13 @@ def main(stop_time, run_simu=True, make_graphs=True):
         cnwheat_hiddenzones_inputs_t0 = pd.read_csv(CNWHEAT_HIDDENZONE_INPUTS_FILEPATH)
         cnwheat_elements_inputs_t0 = pd.read_csv(CNWHEAT_ELEMENTS_INPUTS_FILEPATH)
         cnwheat_soils_inputs_t0 = pd.read_csv(CNWHEAT_SOILS_INPUTS_FILEPATH)
-        update_cnwheat_parameters = {'roots': {'K_AMINO_ACIDS_EXPORT': 3E-5,
-                                               'K_NITRATE_EXPORT': 1E-6}}
+        update_cnwheat_parameters = {'roots': {'K_AMINO_ACIDS_EXPORT': 25*3E-5,

```

Comment:
why these parameters are defined here and not in parameters.py?

Reply:
Some parameters differ between postflowering and vegetative stages. 
If you prefer, I can put both values in parameters.py and pass an option_postflowering_stages to know which on to use.

Reply:
ok let's let it as it is for this NEMA example